nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3760,7 +3764,8 @@ Providing three total network interfaces, including  `nifi.web.http.network.inte
 |`nifi.web.https.port.forwarding`|Same as `nifi.web.http.port.forwarding`, but with HTTPS for secure communication. It is blank by default.
 |`nifi.web.https.ciphersuites.include`|Cipher suites used to initialize the SSLContext of the Jetty HTTPS port.  If unspecified, the runtime SSLContext defaults are used.
 |`nifi.web.https.ciphersuites.exclude`|Cipher suites that may not be used by an SSL client to establish a connection to Jetty.  If unspecified, the runtime SSLContext defaults are used.
-
+|`nifi.web.max.access.token.requests.per.second`| Maximum amount of requests per second the token endpoints will accept before filtering requests. The default value is `25`
+|`nifi.web.should.send.server.version`| Whether the Server header should be included in HTTP responses. The default value is `TRUE` +
```

Comment:
Following the convention of other `boolean` properties, the default value is `true`, not `TRUE`.
```suggestion
|`nifi.web.should.send.server.version`| Whether the Server header should be included in HTTP responses. The default value is `true` +
```